.github/workflows/linter.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ]  You have a great pull request summary, however you should have a descriptive title telling us what changes are inside. (eg." Set up Hello microverse")